source/main.lua

```diff
@@ -302,7 +338,7 @@ function playdate.update ()
     -- title screen
```

Comment:
while not your fault, the `gameState` blocks are not in the correct order. It should be as follows: 
start -> lost -> title -> paused -> playing

this way, all the unusual cases are handled first and there is no need to check the state when writing game logic for `playing`. 

Any check (except for `start` and `playing`, since it will get to the end of the function and return anyways) should return if successful, so that no other code below is run (unnecessary, and keeps away unwanted behavior such as pausing during a menu).